public/validators/main-albatross/NQ37 6EL5 BP9K XL1A 3ED0 L3EC NPR5 C9D3 BRKG.json

```diff
@@ -3,8 +3,14 @@
   "description": "Secure.Reliable.Swiss.",
   "website": "https://helvetia-staking.ch",
   "address": "NQ37 6EL5 BP9K XL1A 3ED0 L3EC NPR5 C9D3 BRKG",
-  "fee": 0.08,
+  "fee": 0.069,
+  "payoutSchedule": "Every 360 batches (~6hrs)"
```

Comment:
This test ideally will be shown to many type of users including beginner users.

Would you mind to rename it just to "Every 6 hours" to keep consistency with the rest of the validators?

thank you!